experiments/calibration/calibrate_land.jl

```diff
@@ -1,5 +1,19 @@
+using Dates # needs to be called again for workers
+
+model = "bucket" # or "bucket"
+const variable_list = ["lhf", "shf"] # variables you want to capture by adjusting your priors
+const n_iterations = 10 # 1 iterations takes ~ 1.5 hour with current settings (resolution, 2 year simulation)
+const spinup_period = Year(1)
```

Comment:
does this mean that we could we make this smaller now, and e.g. run for less time per iteration?

Reply:
e.g. 1.5 years if const spinup_period = 6 month

Reply:
yes!